packages/db/src/resources/networks/index.ts

```diff
@@ -38,13 +38,15 @@ export const networks: Definition<"networks"> = {
         minimumHeight: Int # default any height
         includeSelf: Boolean # default false
         onlyEarliest: Boolean # default false
+        batchSize: Int # default 10
```

Comment:
So `batchSize` is different from `limit`? I see where both are used but my spidey sense tells me to ask about this...

Reply:
`limit` is "how many genealogy iterations", `batchSize` is "how many networks to load from persistence at one time". This is probably worth documenting, but since it's mostly internal, I figure we can ignore for now.